packages/agent-utils/rollup.config.mjs

```diff
@@ -16,7 +16,7 @@ export default {
   plugins: [typescript(), commonjs(), resolve(), {
     renderChunk(code) {
       return `import { globalToShimVarNameMap } from "./shims/globals";
-export const packagesShim = \n\`${code}\`;
+export const packagesShim = \n\`${code.replaceAll("`", "\\`").replaceAll("$", "\\$")}\`;
```

Comment:
What's this about?

Reply:
Search for "\\`" and "\\$" in this file to see why: https://github.com/polywrap/evo.ninja/blob/748c6d333b53bee6c4c71f288a867deece3530e9/packages/agent-utils/src/scripts/js-engine/shims.ts#L176

We need this to ensure the root string doesn't get escaped early by the code within it.